swarm/network/stream/delivery.go

```diff
@@ -21,10 +21,10 @@ import (
 	"errors"
 	"time"
 
-	"github.com/ethereum/go-ethereum/common"
+	"fmt"
```

Comment:
this looks like a linter problem

Reply:
There is no linter error, what is suspicious for you?

Reply:
shouldn't `fmt` come after `errors` and before `time`?